Bunch of cleaning up in tests

jfalcou opened the pull request:
- Remove unused tests
- Get rid of old tests specification in some functions
- Remove remnant of old CMake script that are now useless

jtlap commented:
si c'est juste raw(acos) c'est pas grave : il est prévu pour ne pas donner de (très) bons resultats près de 1.
2 manières de gérer 
  1) utiliser une erreur absolue
 2) réduire l'intervalle de test entre -1 et 0.9

jtlap commented:
note qu'il y a ça aussi : 
 35/80 Test #674: unit.real.core.gather.exe ...........................***Failed    1.95 sec
  [EVE] - Target: eve::avx512_ - Assertions: Enabled  - PRNG Seed: 18102008
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<1> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<2> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<4> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<8> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<16> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<32> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<1> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<2> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<4> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<8> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<16> >)
  [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<signed char, eve::fixed<1> >)
  [EVE] - Assertion 'is_aligned<alignment()>(p)' failed !
  In file /home/falcou/dev/actions-runner/legend-1/_work/eve/eve/include/eve/memory/aligned_ptr.hpp, line 78
  Because:
  0x7ffd4c587670 is not aligned on 64 elements (or 64 bytes).

jfalcou commented:
> note qu'il y a ça aussi : 35/80 Test #674: unit.real.core.gather.exe ...........................***Failed 1.95 sec [EVE] - Target: eve::avx512_ - Assertions: Enabled - PRNG Seed: 18102008 [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<1> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<2> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<4> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<8> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<16> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<float, eve::fixed<32> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<1> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<2> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<4> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<8> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<double, eve::fixed<16> >) [SCENARIO] - Check eve::gather behavior with 32 bits indexes (with T = eve::avx512_abi_v0::wide<signed char, eve::fixed<1> >) [EVE] - Assertion 'is_aligned<alignment()>(p)' failed ! In file /home/falcou/dev/actions-runner/legend-1/_work/eve/eve/include/eve/memory/aligned_ptr.hpp, line 78 Because: 0x7ffd4c587670 is not aligned on 64 elements (or 64 bytes).

I know. I probably replaced a T with v_t instead of wide<T> somewhere.
